### ISSUE TITLE
Feature Request -- Backup option for yaml, configs, volumes etc

### ISSUE THREAD

**Issue by bd8392** [swarmpit/swarmpit#462]
---------------------------------------------------
FEATURE REQUEST
---------------------------------------------------

Hi =) I LOVE swarmpit! it is very intutive, and you're able to config swarm very easy! The only two options that are missing, are (incremental)backup and download options for configs, secrets, yamls, volumes etc. Then it would be perfect! =) Thanks for all the development =)

**Comment by mrspartak** [comment]
Any updates on this? Could be very useful for migrating to another provider or if everything dies to redeploy fast.

**Comment by lumir-mrkva** [comment]
i agree, this feature is needed. i could just export the complete swarm state as a collection of stackfiles. most complex part will be export and import of config/secret, which is not supported currently at all.
